apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/AspNetCoreSnippets.cs

```diff
@@ -81,35 +81,33 @@ public void ReadFile(IExceptionLogger exceptionLogger)
         // End sample
 
         // Sample: RegisterGoogleLogger
-        public void Configure(ILoggerFactory loggerFactory)
+        public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
         {
             string projectId = "[Google Cloud Platform project ID]";
-            loggerFactory.AddGoogle(projectId);
+            loggerFactory.AddGoogle(app.ApplicationServices, projectId);
         }
         // End sample
 
-#if NETCOREAPP2_0
         private class Startup { }
-        public void Configure()
+        public static void Program_Main()
```

Comment:
Why the change here?

Reply:
The reason was that `Configure` implies that the code is located in the startup class. Anyhow, I could revert it.

Reply:
Fair enough.  Just more curious than anything. The users will won't see this, they will only see things between the `// Sample: ` bits.